pytorch_lightning/trainer/trainer.py

```diff
@@ -474,7 +482,7 @@ def __init__(
             self.show_progress_bar = show_progress_bar
 
         self.progress_bar_refresh_rate = progress_bar_refresh_rate
-        self.progress_bar_callback = None
+        self.progress_bar_callback = progress_bar_callback
```

Comment:
the arg `progress_bar_callback` was not used anywhere - forgotten, hope it is the right place...